akka-docs/rst/scala/code/docs/routing/RouterDocSpec.scala

```diff
@@ -83,6 +84,25 @@ akka.actor.deployment {
   }
 }
 #//#config-balancing-pool2
+
+#//#config-balancing-pool3
+akka.actor.deployment {
+  /parent/router10b {
+    router = balancing-pool
+    nr-of-instances = 5
+    pool-dispatcher {
+      executor = "thread-pool-executor"
+
+      # allocate exactly 5 threads for this pool
+      thread-pool-executor {
+        core-pool-size-factor = 2.0
```

Comment:
this can be removed, doesn't do anything when min and max are given to the same value